.github/workflows/crowdin-download.client-ui.yml

```
@@ -238,6 +238,35 @@ jobs:
           # Uncomment below to debug
           # dryrun_action: true
 
+      ##### Download Arabic #####
+      - name: Crowdin Download Arabic Translations
+        uses: crowdin/github-action@master
+        # options: https://github.com/crowdin/github-action/blob/master/action.yml
+        with:
+          # uploads
+          upload_sources: false
+          upload_translations: false
+          auto_approve_imported: false
+          import_eq_suggestions: false
+      
+          # downloads
+          download_translations: true
+          download_language: arabic
```

```suggestion
          download_language: ar
```

This needs to be the two character ISO code.